lms/db/__init__.py

```diff
@@ -104,20 +104,6 @@ def init(engine, drop=False):
     BASE.metadata.create_all(engine)
 
 
-def _stamp_db(engine):  # pragma: nocover
-    """
-    Stamp the database with the latest revision if it isn't already stamped.
-
-    This is convenient in development environments to automatically stamp a new
-    database after initializing it.
-
-    """
-    try:
-        engine.execute("select 1 from alembic_version")
```

Comment:
Not stamping if there's already an `alembic_version` table means that `create_all` would create all new tables but alembic, which we run on `make db` wouldn't know, (alembic_version didn't change) and would try to create itself.